test/school-service.test.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+var chai = require('chai');
+var assert = chai.assert;
+var _ = require('lodash');
+var SchoolServiceFactory = require('../src/school-service');
+var RegistrationFormFactory = require('../src/registration-form');
+var UuidFactory = require('../src/uuid-generator');
+
+describe('Testing school-service', function () {
+
+    describe('for valid creation', function () {
+
+        it('should return a school service', function () {
+            var schoolService = SchoolServiceFactory.create();
+
+            assert.isNotNull(schoolService);
+        });
+    });
+
+    describe('create new school', function () {
+
+        var schoolRepositoryStub = {
```

Comment:
test code duplication - please separate to different function - and get NEW INSTANCE of schoolRepositoryStub!!